lib/src/coap_client.dart

```diff
@@ -205,20 +205,21 @@ class CoapClient {
     required final String payload,
     final CoapMediaType format = CoapMediaType.textPlain,
     final CoapMediaType accept = CoapMediaType.textPlain,
-    final int type = CoapMessageType.con,
+    final bool confirmable = true,
+    final CoapMessageType type = CoapMessageType.con,
```

Comment:
Remove this for `put` since we have the `confirmable` bool now, guessing this is a typo? Could be confusing if `confirmable` is `false` and `type` is `CoapMessageType.con` for example.

Reply:
Thank you, that was actually a typo :)